main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -248,6 +249,11 @@ class Flix {
     */
   val genSym = new GenSym()
 
+  /**
+    * A class loader for loading JARs.
```

Comment:
=> loading external jars

(I am going to suggest the word "external" to mean JARS that are not part of the Flix classpath.)